alvr/xtask/src/version.rs

```diff
@@ -70,6 +70,17 @@ fn bump_cargo_version(crate_dir_name: &str, new_version: &str) {
     fs::write(manifest_path, manifest).unwrap();
 }
 
+fn bump_spec_version(new_version: &str) {
+    let spec_path = crate::workspace_dir().join("packaging/rpm/alvr.spec");
```

Comment:
`workspace_dir` is used in two places in this file now. Can you `use` it?

Reply:
Fixed